src/app/shared/grammars/custom-grammars.js

```diff
@@ -158,7 +159,7 @@ ace.define('ace/mode/cwl', ['require', 'exports', 'module', 'ace/lib/oop', 'ace/
   exports.Mode = Mode;
 });
 
-// WDL Grammar
+// WDL Grammar generated originally from the tmLanguage of https://github.com/broadinstitute/wdl-sublime-syntax-highlighter using https://github.com/ajaxorg/ace/wiki/Importing-.tmtheme-and-.tmlanguage-Files-into-Ace#importing-textmatesublime-languages
```

Comment:
Yikes, that tmLanguage is 4 years old. But they say it support WDL 1.0, so it seems good.